core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -77,6 +77,8 @@ public final class VirtualHost {
     private final String defaultHostname;
     private final String hostnamePattern;
     private final int port;
+    private final String contextPath;
```

Comment:
I don't think we need to add `contextPath` to `VirtualHost` for the new direction.
What do you think of reverting this file?